internal/yunikorn/stream.go

```diff
@@ -73,7 +64,18 @@ func (s *Service) processStreamResponse(ctx context.Context, response []byte) er
 		"change_detail", eventRecord.GetEventChangeDetail(),
 		"reference_id", eventRecord.GetReferenceID(),
 		"resource", eventRecord.GetResource(),
+		"state", eventRecord.GetState(),
 	)
 
+	// TODO: This is Okayish for small number of events, but for large number of events this will be a bottleneck
```

Comment:
Can you add a ticket instead of a TODO?

Reply:
I don't think we should do anything about it unless we have evidence it is a bottleneck. We are not doing something complicated, so I would avoid implementing parallel processing for now